src/sentry/api/urls.py

```diff
@@ -1906,6 +1907,11 @@
             ]
         ),
     ),
+    re_path(
+        r"^(?P<organization_slug>[^/]+)/metrics/code-locations/$",
```

Comment:
Should we instead create an endpoint that exposes all kinds of meta data stored with the metrics (knowing that "meta" is taken)? The system is currently built to allow different kinds of meta data, and I think we don't want to build a new endpoint for each of those.

Reply:
How about having it like [5b7a4ac](https://github.com/getsentry/sentry/pull/60369/commits/5b7a4acaa9524e6c5aa413d23983b8dfdff4919a) for now?